### ISSUE TITLE
Multiple properties 

### ISSUE THREAD

**Issue by skacco** [oc-shopaholic/oc-shopaholic-plugin#311]
Hi,

I need to have 2 select dropdown (or radio button) to choose from and update the price accordingly.

For example, a jeans offer would have a Size and a Color properties.
Size: S, M, L, XL
Color: Blue, Black, Red
Each combination has his own price.

on the product page: 2 select dropdown, one to select the size and the other one to select the Color.
upon the choice of the size and the color, the price would be updated accordingly on the product page.

I have already create offers for all possible combinations of colors and sizes.
I have show 2 select with possible variants of properties. 
I need to have the ajax code to update pricing, photos and others.

**Comment by kharanenka** [comment]
Hi! We do not have ready-made solution for implementing such complex logic of choice. We usually make products of different colors as related products. It's better for SEO and easier to implement.

You can display prices for combinations in attributes or json objects. You can write your own custom component that will return response in json format that you need, receiving OfferItem object

**Comment by brahmbhattsnehal** [comment]
@kharanenka @skacco Would you happen to have any solutions that could help with this issue?

**Comment by kharanenka** [comment]
Look at this [page](http://atmosphere.lovata.tech/catalog/decor/candles-candleholders/swirl-candle-set-1000026655). Does the behavior look like what you expect?

**Comment by brahmbhattsnehal** [comment]
@kharanenka Yes, I’d like to implement the same behavior, but with multiple attributes affecting the price. Would you be able to share the code, please?
